README.md

```diff
@@ -1,3 +1,17 @@
 Data Together
 ========
 > Community Patterns for the Decentralized Web
```

Comment:
Feel like we should update this header a wee bit (at least the styling!)

Reply:
haha sure! It was in there from before, so i just left it.